spec/lib/lighthouse/facilities/client_spec.rb

```diff
@@ -0,0 +1,151 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Lighthouse::Facilities::Client do
+  let(:params) do
+    {
+      bbox: [60.99, 10.54, 180.00, 20.55] # includes the Phillipines and Guam
```

Comment:
Does this need a bounding box? the current facilities lookup gets a center point and radius.

Reply:
Doesn't the facilities locator use a bounding box when you search for a state, city, or zip?

Reply:
PPMS yes, but for VA facilities no

Reply:
I have added the distance from the requested point to facilities when that data is returned.